app/src/androidTest/java/com/example/henzoshimada/feeltrip/FeelTripTests.java

```diff
@@ -67,12 +67,11 @@ public void testGetMoodTask(){
         assertEquals("test getMood", "user1", moods.get(0).getUser());
     }
 
-    // does not pass
     public void testEditTask() throws DescriptionTooLongException {
```

Comment:
Tested and working with edits on all fields except user (but really, we won't be changing the user)